src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+
+import { Sorting } from '../../types/Sorting';
+
+type Props = {
+  sortBy: Sorting,
+  setSortBy: (sort: Sorting) => void,
+  todosCount: number,
+  onRemove: () => void,
+  complitedTodos: number[],
+};
+
+export const Footer: React.FC<Props> = ({
+  sortBy,
+  setSortBy,
+  todosCount,
+  onRemove,
+  complitedTodos,
+}) => (
+  <footer className="todoapp__footer" data-cy="Footer">
+    <span className="todo-count" data-cy="todosCounter">
+      {`${todosCount} items left`}
+    </span>
+
+    <nav className="filter" data-cy="Filter">
+      <a
+        data-cy="FilterLinkAll"
+        href="#/"
```

Comment:
why do you use `#` & `/` together?

Reply:
this markup href="#/" has already been existed in the task by default